data/human/human missions.txt

```diff
@@ -4440,3 +4440,320 @@ mission "Dangerous Games 1"
+			`	You goggle at the display for a moment, taking it in, and draw breath to call out the alarm when an eye appears through the ship window, yellow, glowing, alien, and massive.`
+			label run
+			`	All at once, you turn and accelerate, shouting the alarm to any who hadn't already realized, and hoping that you're leaving the leviathan beasts behind. A quick glance at the display shows at least one subpod following behind. Good enough. If one of those rams the ship, nobody is going home anyway.`
+			`	It's a long while before you slow and turn around. No mammoth sea monster comes out of the darkness, and you take a breath.`
+			`	"Quoijuu," says Karengo simply.`
+				goto failure
+			label east5
+			action
+				set "Jorma and Daciana dead"
+			`	Small schools of fish flit past as you guide your small fleet to the west. Minutes pass with no sign of danger, and finally Karengo broadcasts, "Good work, <first>. Looks like this is the way forward."`
+			`	"Last one there's a rotten fish eye!" shouts Jorma, and your screen shows her subpod leap forward and gain on you, followed quickly by Daciana's.`
+			`	Then, without warning, a proximity sensor goes off at the same time your outer lights show a wall directly in front, stretching for as far as you can see. "Stop!" shouts Karengo, "It's the end of the cavern! Stop and go back!"`
+			`	Shuddering, your ship comes to a halt mere meters from the endless stone face. With a sinking feeling that has nothing to do with the depth, you turn the ship around and head back the way you came.`
+				goto failure
+			label dead2
+			choice
+				`	(Go north.)`
+					goto north3
+				`	(Go east.)`
+					goto east3
+				`	(Go south.)`
+			label uninterested
+			`	You smoothly turn the ship to the right, heading south.`
+			`	"Should be close," radios Karengo. "Fire off a ping." It's almost a relief to send out a pulse to map the surroundings after flying dark for so long.`
+			`	The result, however, is a surprise. The display shows a massive object directly below you... as well as as two leviathan-sized things back the way you came, each over 200 meters long. Dozens of sets of flippers run along each 80-meter-tall side and hundreds of teeth the size of an adult human line their jaws.`
+			`	You goggle at the display for a moment, holding your breath, but the leviathans seem uninterested. You shudder, imagining what one of those could do if you'd turned the wrong way, before your interest returns to the mysterious bulk resting below the ship.`
+				goto treasure
+			label north3
+			action
+				set "Jorma and Daciana dead"
+			`	Disheartened at this point, you make the call to go north, completing the zig-zag pattern.`
+				goto subpods
+			label east3
+			action
+				set "Jorma and Daciana dead"
+			`	By now, the feeling that you get when the ship's subpod tracker loses two more dots is familiar. Jorma and Daciana are lost to the eastern wall, leaving a single remaining subpod: Karengo's.`
```

Comment:
This line is followed by `Jorma and Daciana never report back in` (line 4673), which doesn't really make sense considering this line already mentioned they're gone. Renovating line 4673 doesn't make sense because so many other scenarios lead us there, so I think this one should be edited instead. You have any ideas on what we should replace this with?

Reply:
Thanks for catching this! I changed the line a bit, let me know what you think.